htlcswitch/link.go

```diff
@@ -2330,6 +2330,23 @@ func (l *channelLink) processRemoteAdds(fwdPkg *channeldb.FwdPkg,
 				continue
 			}
 
+			// Reject htlcs for canceled invoices.
+			if invoice.Terms.State == channeldb.ContractCanceled {
+				log.Errorf("Rejecting htlc due to canceled " +
```

Comment:
Should use `l.errorf` instead so that the short channel id is prepended to the log message

Reply:
Fixed this, but noticed that in many other places `l.errorf` isn't used. Also, I think error is not the right log level for something that can just happen normally. But don't want to tackle that in this pr.